src/index.js

```diff
@@ -15,21 +21,23 @@ Promise.all([roomsData, usersData, bookingsData])
     roomsData = data[0];
     usersData = data[1];
     bookingsData = data[2];
-  })
-  .then(() => {
-    checkLogin(roomsData, usersData, bookingsData);
-  })
-
-function checkLogin(roomsData, usersData, bookingsData) {
-  console.log(roomsData);
-  console.log(usersData);
-  console.log(bookingsData);
+  }).then(() => {
+    $('.explore-button').click(function() {
+      rooms = new Rooms(roomsData, bookingsData)
+      checkLoginInfo(roomsData, usersData, bookingsData);
+    })
+})
+
+function checkLoginInfo(roomsData, usersData, bookingsData) {
+  let userId = parseInt($('#username-input').val().slice(8));
```

Comment:
what happens if it's another 8 character word? It seems like it would still let them login.

Reply:
farthead50